api/database/seeders/BigSeederPoolCandidateUser.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Enums\AssessmentResultType;
+use App\Enums\AssessmentStepType;
+use App\Enums\PoolSkillType;
+use App\Models\AssessmentResult;
+use App\Models\AssessmentStep;
+use App\Models\Community;
+use App\Models\Pool;
+use App\Models\PoolCandidate;
+use App\Models\PoolSkill;
+use App\Models\User;
+use Illuminate\Database\Seeder;
+
+class BigSeederPoolCandidateUser extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * This seeds a lot of data
+     * Run this AFTER core data has been seeded, this will not seed platform data
+     *
+     * User/PoolCandidate/Pool/AssessmentResult
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $input = $this->command->ask('Please enter how many times to loop');
+        $limit = intval($input);
+        $limit = (is_int($limit) && $limit > 0) ? $limit : 1;
+
+        // constant values for reuse and setup
+        $digitalCommunityId = Community::select('id')->where('key', 'digital')->sole()->id;
+        $atipCommunityId = Community::select('id')->where('key', 'atip')->sole()->id;
+        $financeCommunityId = Community::select('id')->where('key', 'finance')->sole()->id;
+
+        $pools = Pool::factory()
+            ->count(2)
+            ->withPoolSkills(4, 4)
+            ->withQuestions(2, 2)
+            ->published()
+            ->withAssessments(3)
+            ->create();
+        $poolIds = $pools->pluck('id')->toArray();
+
+        // User - non-government
+        for ($i = 0; $i < $limit; $i++) {
+            User::factory()
+                ->asApplicant()
+                ->withSkillsAndExperiences()
+                ->afterCreating(function (User $user) use ($poolIds) {
+                    $this->applyToPools($user, $poolIds);
+                })
+                ->count(100)
+                ->create();
+        }
+
+        // User - government
+        for ($i = 0; $i < $limit; $i++) {
+            User::factory()
+                ->asApplicant()
+                ->withSkillsAndExperiences()
+                ->asGovEmployee()
+                ->withEmployeeProfile()
+                ->withCommunityInterests([
+                    array_rand(array_flip([$digitalCommunityId, $atipCommunityId, $financeCommunityId])),
+                ])
+                ->afterCreating(function (User $user) use ($poolIds) {
+                    $this->applyToPools($user, $poolIds);
+                })
+                ->count(100)
+                ->create();
+        }
+
+        // AssessmentResult - Pool one
+        $poolOne = $poolIds[0];
+        $assessmentSteps = AssessmentStep::where('pool_id', $poolOne)->get();
+        $essentialPoolSkills = PoolSkill::where('pool_id', $poolOne)->where('type', PoolSkillType::ESSENTIAL->name)->get();
+        $poolOneCandidatesToFillResults = PoolCandidate::where('pool_id', $poolOne)
+            ->inRandomOrder()
+            ->take(100)
+            ->get();
```

Comment:
Convenient to generate some `AssessmentResult` models while working with related models. But to avoid expanding the scope or seeder too much few, relatively speaking, candidates will have results. 
I think it is important to generate some for the time being, but expanding seeding down the line can go further

Reply:
It takes really long time to create the poolcandidate users in dev  which terminates the shell and doesn't run the AssessmentResult section. 
I'm running it with only 100 candidates now .. let me see if it can finish